main.go

```
@@ -0,0 +1,73 @@
+	// Unsafe file operations
+	data, _ := os.ReadFile("/etc/secrets/api_key")
+	return string(data)
+}
+
+// Weak password validation
+func isPasswordStrong(password string) bool {
+	return len(password) >= 6
+}
+
+// Insecure random number generation
+func generateToken() string {
```

golangci-lint (1.62.2)

func `generateToken` is unused

(unused)